xchange-stream-btcmarkets/src/main/java/info/bitrich/xchangestream/btcmarkets/BTCMarketsStreamingAdapters.java

```diff
@@ -80,14 +80,9 @@ public static Trade adaptTradeMessageToTrade(BTCMarketsWebSocketTradeMessage mes
 
   private static OrderType getOrderType(String side) {
     OrderType orderType = null;
-    switch (side) {
-      case "Ask":
-        orderType = OrderType.ASK;
-      case "Bid":
-        orderType = OrderType.BID;
-      default:
-        LOG.info("Trade side {} not recognised, set to null", side);
-    }
+    if (side.compareToIgnoreCase("ask") == 0) orderType = OrderType.ASK;
```

Comment:
this is not what i meant i mean
```
case "Ask":	    
        return OrderType.ASK;
```

Reply:
Just use BTCMarketsAdapters.adaptOrderType , it's there already.